README.md

```diff
@@ -6,8 +6,8 @@ Amplify your team's potential with customizable and secure AI assistants.
 
 ## :book: Developer Documentation
 
-- [LLM apps Primer](https://docs.dust.tt/introduction)
-- [Developer Platform overview](https://docs.dust.tt/overview)
-- [Blocks documentation](https://docs.dust.tt/core-blocks)
-- [Getting started guide](https://docs.dust.tt/quickstart)
-- [Runs API Reference](https://docs.dust.tt/runs)
+- [LLM apps Primer](https://docs.dust.tt/docs/prompting-101-how-to-talk-to-your-assistants)
```

Comment:
This one should be removed :+1: 

Then happy to merge!

Reply:
Line removed. Thanks for your feedback @spolu